backend/api/appsettings.json

```diff
@@ -22,14 +22,22 @@
   },
   "AllowedHosts": "*",
   "Pims": {
+    "AccessRequest": {
```

Comment:
Provides a configurable way to identify the notification template that will be used by access requests.